usaspending_api/api_contracts/contracts/v2/recipient/children.md

```diff
@@ -0,0 +1,39 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Recipient Profile
+
+This endpoint is used to power USAspending.gov's recipient profile pages. This data can be used to visualize the government spending that pertains to a specific recipient.
+
+# Group Profile Page
```

Comment:
Since we're moving each endpoint to a separate file, we could get rid of the "Group" heading and keep just a  heading/ description specific to this file. (Similar elsewhere)

Reply:
Agreed. I wasn't certain on what to do. Can do that on my next pass.